src/test/java/com/wikia/webdriver/testcases/featuredVideo/FeaturedVideoDesktopTests.java

```diff
@@ -0,0 +1,130 @@
+package com.wikia.webdriver.testcases.featuredVideo;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00120)

Reply:
please rename this package name to `com.wikia.webdriver.testcases.featuredvideo;`